### PR TITLE
Adding the One sample t-test for Python

### PR DESCRIPTION
**Pull request by LukasBrausch**
This PR aims to introduce the One sample t-test for Python.

### CODE REVIEW

File: python/OneSampleTTest.qmd [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "One Sample t-test"
+---
+
+---
+subtitle: "One Sample t-test in Python"
+---
```
Comment (statasaurus) [inline python/OneSampleTTest.qmd:7]: The part outside of the code blocks just follow traditional markdown notation so like the following 
```suggestion
## One Sample t-test in Python
```

File: python/OneSampleTTest.qmd [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+3.*nan_policy*: Defines how to handle input NaNs.
+
+4.*alternative* (optional): Defines the alternative hypothesis.
+
+5.*keepdims*: If this is set to True, the axes which are reduced are left in the result as dimensions with size one. With this option, the result will broadcast correctly against the input array.
+
+---
+subsubtitle: "Data Used"
+---
```
Comment (statasaurus) [inline python/OneSampleTTest.qmd:24]: See above 
```suggestion
## Data Used

```

**Review by statasaurus — CHANGES_REQUESTED** [review]
This looks really good @LukasBrausch,  I just have some minor formatting requests. But, it looks like you were able to get the requirements updated so that is great to see

**Comment by LukasBrausch** [comment]
Thank you very much for the suggestions, @statasaurus. I accepted them just now. Once I get a feeling for how things work around here, I hope to be able to commit some further Python comparisons. :)

Have a great day.

**Review by statasaurus — APPROVED** [review]
This looks great thank you!

**Pull request merged**